addons/full-signature/disable.js

```diff
@@ -0,0 +1,12 @@
+export default async function ({ addon, console }) {
+  addon.self.addEventListener("disabled", () => {
+    document
+      .querySelectorAll(
+        ".activity-stream, .activity-ul, .postsignature, .project-title.no-edit, .scratchblocks, .blocks3"
+      )
+      .forEach((el) => {
+        el.scrollTop = 0;
+        el.scrollLeft = 0;
```

Comment:
Will this need RTL support?

Reply:
Scratch only supports RTL layout in the editor.

Reply:
I changed a project's title to Arabic text using Inspect and only the *end* of the title was visible, even if I set the language to Arabic.